deploy/helm_charts/envs/custom.yaml

```diff
@@ -54,3 +54,6 @@ nl:
 
 customDC:
   enabled: true
+
+svg:
+  blocklistFile: ["dc/g/Uncategorized", "dc/g/SDG", "sdg/g/SDG"]
```

Comment:
So this blocks 3 svg from all the custom DC (including Stanford). I wonder what is the plan to update this later on?

Reply:
"dc/g/Uncategorized" was always blocked in all staging/production instances of the website & in terms of the sdg groups being blocklisted, this change was requested in order to block those from showing up on the hierarchy for Stanford DC .. so not sure what the future plan is?